Mobile/ShopLens/Droid/Source/AppPreferences.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using Android.Content;
+
+namespace ShopLens.Droid.Source
+{
+    public class ActivityPreferences
+    {
+        ISharedPreferences prefs;
+        ISharedPreferencesEditor prefsEditor;
+        Context pContext;
+        List<string> items;
+        int itemCounter = 0;
+
+        public ActivityPreferences(Context context, string name)
+        {
+            pContext = context;
+            prefs = pContext.GetSharedPreferences(name, FileCreationMode.Private);
+            prefsEditor = prefs.Edit();
+            foreach (KeyValuePair<string, object> entry in prefs.All)
```

Comment:
Can't open the IDE right now, but isn't there a simpler way to do this? Like `itemCounter = prefs.Count` or something like that?

Reply:
Fixed.